xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/test/java/org/xwiki/tool/checkstyle/UnstableAnnotationCheckTest.java

```diff
@@ -35,15 +32,15 @@
  * @version $Id$
  * @since 8.1M1
  */
-public class UnstableAnnotationCheckTest extends BaseCheckTestSupport
+public class UnstableAnnotationCheckTest extends AbstractModuleTestSupport
 {
     private DefaultConfiguration checkConfig;
 
     @Before
     public void setUp()
     {
         this.checkConfig = createCheckConfig(UnstableAnnotationCheck.class);
-        this.checkConfig.addAttribute("currentVersion", "8.1-SNAPSHOT");
+        this.checkConfig.addAttribute("currentVersion", "8.2-SNAPSHOT");
```

Comment:
I don't think you need to change this version because it's not the Checkstyle version.